testsuite/ui/views/admin/backend/__init__.py

```diff
@@ -52,7 +52,7 @@ class BaseBackendView(BaseAdminView):
     """
 
     NAV_ITEMS = ["Overview", "Analytics", "Methods and Metrics", "Mapping Rules"]
-    nav = NavigationMenu(id="mainmenu")
+    nav = NavigationMenu(component_id="OUIA-Generated-Nav-1")
```

Comment:
Comment for next time, not now. There should be something like:

```
class ThrescaleNavigationMenu:  # name should be completely different, not this loooong tragedy, I only wanted to differentiate it from generic NavigationMenu (if appropriate, the name can be NavigationMenu also here)
    def __init__(self, component_id="OUIA-Generated-Nav-1"):
      ...
```
This will prevent duplicate lines and will make this change smaller and simpler. Obviously it is not bound to NavigationMenu, it is applicable generally.